src/components/pages/Reservations.jsx

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable */
+import './../pages/styles/reservations.css'
+import React, { useEffect} from 'react'
+import { useSelector } from 'react-redux';
+
+
+const Reservations = ({ reservationId }) => {

```

Comment:
- [ ] When I try to add a reservation, this happens:

![image](https://user-images.githubusercontent.com/57421823/227748954-2fe606a4-6c8e-4d84-8f20-d1f57319baae.png)

I get redirected to the "Reservations" page and nothing shows up.

It seems currentUser is undefined, there seems to be an issue in the Redux implementation in which the user is not being persisted.

The server is never reached, and the database suffers no changes (no new reservation record):

![image](https://user-images.githubusercontent.com/57421823/227749085-f23d29a7-5caa-48ed-a880-e8381ff5b6f8.png)


For context, if I try to add a console log after the user state is being called:

![image](https://user-images.githubusercontent.com/57421823/227749057-45f380eb-403a-4563-b3eb-bf218dd5bc81.png)

It returns undefined.

You are very close, kindly fix this bug so your users can make reservations. 👍